src/proxy/health.rs

```diff
@@ -38,7 +38,7 @@ impl Health {
         let healthy = health.healthy.clone();
         let default_hook = panic::take_hook();
         panic::set_hook(Box::new(move |panic_info| {
-            error!(log, "Panic has occurred. Moving to Unhealthy");
+            error!(log, "Panic has occurred. Moving to Unhealthy: {}", panic_info);
```

Comment:
Moving this to a structured format, so it's easier to analyse (code was off the top of my head, may not be 100% correct).

```suggestion
            error!(log, "Panic has occurred. Moving to Unhealthy"; info => format!(panic_info));
```